src/styles/blocks/explore.scss

```diff
@@ -0,0 +1,56 @@
+.explore {
+  &__item {
+    margin-bottom: 48px;
+
+    &:last-child {
+      margin-bottom: 0;
+    }
+
+    @media onTablet {
+      margin-bottom: 0;
+    }
+  }
+}
+
+.explore-option {
+  &__photo {
+    width: 100%;
+    object-fit: center;
```

Comment:
object-fit hasn't such property
https://developer.mozilla.org/ru/docs/Web/CSS/object-fit

Reply:
thank you for the correction, it have been fixed